src/test/java/qna/domain/AnswerTest.java

```diff
@@ -1,6 +1,66 @@
 package qna.domain;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
 public class AnswerTest {
     public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
     public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+    @Autowired
+    private AnswerRepository answers;
+
+    @Test
+    void save() {
+        // when
+        final Answer actual = answers.save(A1);
+
+        //then
+        assertThat(answers.findById(actual.getId()).get()).isEqualTo(actual);
```

Comment:
ID가 부여되었는지 확인하는 테스트 코드를 작성해도 충분할 것 같아요!

```suggestion
        assertThat(actual.getId()).isNotNull();
```

Reply:
생성에 대한 테스트인데 들어간 값이 정확히 제가 넣은 값인지 확인을 해봐야 하는 것 아닐까요? 아니면 저정도만 확인해도 충분한 걸까요?